src/lib/checkbox/checkbox.spec.ts

```diff
@@ -10,6 +10,32 @@ import {FakeViewportRuler} from '../core/overlay/position/fake-viewport-ruler';
 describe('MdCheckbox', () => {
   let fixture: ComponentFixture<any>;
 
+  /** Creates a DOM mouse event. */
+  const createMouseEvent = (eventType: string, dict: any = {}) => {
```

Comment:
Can you add a TODO to use the testing utils being added in #2943 once they're available